opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""
+.. Explicitly document private _AggregationFactory
+.. autoclass:: _AggregationFactory
```

Comment:
This is needed for the docs to build.

Alternatives:
- Make `_AggregationFactory` public.
- Make a public `AggregationFactory = Union[SumAggregation, HistogramAggregation, DropAggregation, ...]` to avoid even making a public interface for this.

Reply:
The implemented solution is right, does not need any workaround. If we are having a doc-building issue we should fix it with doc-related tools, not by changing the API.